R/registry.R

```diff
@@ -240,6 +241,48 @@ parse_registered_models <- function(registered_models) {
     as_tibble()
 }
 
+
+#' Get a registered model run id
+#'
+#' @param experiment_name An experiment name.
+#' @param client An MLFlow client. Will be auto-generated if omitted.
+#' @param stage A model stage. Set to `NULL` to return all results.
+#' @importFrom purrr pluck
+get_registered_model_run_id <- function(experiment_name, client = mlflow_client(), stage = "Production") {
+
```

Comment:
I think for mlflow purposes, this arg should _technically_ be called `name` or `model_name` or `registered_model_name` . mlflow doesn't force your models to have the same name as your experiments, we're just doing that because, well, it makes no sense to do it any other way

Reply:
i see that now. i think `model_name` is best here